lib/languages.ts

```diff
@@ -725,6 +725,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    gimple: {
+        name: 'GIMPLE',
+        monaco: 'nc',
+        extensions: ['.c'],
+        alias: [],
+        logoUrl: 'gimple.svg',
+        logoUrlDark: null,
+        formatter: 'clangformat',
+        previewFilter: /^\s*#include/,
```

Comment:
Not sure you need to filter `#include`

Reply:
I'm not sure either, I just copied from the config for C